pkg/keychain/keychain_darwin.go

```diff
@@ -9,16 +9,44 @@ import (
 	"github.com/keybase/go-keychain"
 
 	"github.com/confluentinc/cli/v3/pkg/log"
-	"github.com/confluentinc/cli/v3/pkg/netrc"
 )
 
+type Machine struct {
+	Name     string
+	User     string
+	Password string
+}
+
+type MachineParams struct {
```

Comment:
I'm not sure this belongs here. I think this struct is used by the configuration file *and* the keychain, so putting it in the `keychain` package might be misleading?

Reply:
> //go:build darwin

Also, be careful, this file only gets built on macOS.

Reply:
Good catch, thanks.